.github/workflows/build_linux_pytorch_wheels.yml

```
@@ -67,12 +73,12 @@ jobs:
           echo "Building PyTorch wheels for ${{ inputs.PYTORCH_TARGET_ARCH }}"
           ./external-builds/pytorch/build_prod_wheels.py \
             --pip-cache-dir /tmp/pipcache \
-            --find-links "${{ inputs.find_links }}" \
+            --find-links "https://${{ inputs.s3_python_bucket }}.s3.us-east-2.amazonaws.com/${{ inputs.AMDGPU_FAMILIES }}/index.html" \
```

```suggestion
            --find-links "https://${{ env.S3_BUCKET }}.${{ env.S3_ENDPOINT}}/${{ inputs.AMDGPU_FAMILIES }}/index.html" \
```

You added an environment variable thus I would use that instead of the input variable for consistency.